torchmetrics/metric.py

```diff
@@ -359,7 +359,7 @@ def reset(self):
             else:
                 setattr(self, attr, [])
 
-    def clone(self):
+    def clone(self) -> "Metric":
```

Comment:
if you do `from __future__ import annotations` at the top then you can drop the quotes.

Reply:
lets leave it to another PR as it will be in multiple places